recipes/src/main/resources/quarkus-updates/core/3.3.yaml

```diff
@@ -0,0 +1,33 @@
+#####
+# Adjust properties in application.properties
+#####
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: io.quarkus.updates.core.quarkus33.ApplicationProperties
+applicability:
+  singleSource:
+    - org.openrewrite.FindSourceFiles:
+        filePattern: "**/application*.properties"
+recipeList:
+  - org.openrewrite.properties.ChangePropertyKey:
+      oldPropertyKey: quarkus.hibernate-search-orm.automatic_indexing.synchronization.strategy
+      newPropertyKey: quarkus.hibernate-search-orm.indexing.plan.synchronization.strategy
```

Comment:
I guess we don't support named persistence units for upgrades? Because this will ignore them AFAICS.

Maybe with some regexp with capturing groups we could handle named persistence units?